tdp/cli/utils.py

```diff
@@ -72,13 +72,23 @@ def dry(func: FC) -> FC:
     )(func)
 
 
-def mock_deploy(fun: FC) -> FC:
+def env(func: FC) -> FC:
```

Comment:
This option is used once no ? If this is use once, no need to put it here.

Reply:
True, this fonction is not even used anymore, I forgot to removed it.

Reply:
Done